README.md

```diff
@@ -22,34 +22,52 @@ Also, it writes some output into tmp/output.txt file:
+ruby_app           | 2017-10-09T19:25:52.872Z 1 TID-grm1c8lps INFO: Starting processing, hit Ctrl-C to stop
+ruby_app           | 2017-10-09T19:25:52.902Z 1 TID-grm1gpn28 RubyApp::Workers::ProducerWorker JID-384705f769a896f6db0c2519 INFO: start
+ruby_app           | 2017-10-09T19:25:52.932Z 1 TID-grm1gpn28 RubyApp::Workers::ProducerWorker JID-384705f769a896f6db0c2519 INFO: done: 0.03 sec
+ruby_app           | 2017-10-09T19:25:52.936Z 1 TID-grm1gpmns RubyApp::Workers::ProducerWorker JID-7f7b4f804fa65b28ed54e6a5 INFO: start
+ruby_app           | 2017-10-09T19:25:52.938Z 1 TID-grm1gpmns RubyApp::Workers::ProducerWorker JID-7f7b4f804fa65b28ed54e6a5 INFO: done: 0.002 sec
+ruby_app           | 2017-10-09T19:25:53.943Z 1 TID-grm1gy94g RubyApp::Workers::ProducerWorker JID-a1478d317c52a3f5c9306bec INFO: start
+crystal_app        | 2017-10-09T19:25:53.943Z 1 TID-j75z4  JID=528ebf94e23c6e3591c27a1e INFO: Done: 1.005546 sec
+crystal_app        | 2017-10-09T19:25:53.945Z 1 TID-j75vk  JID=4617f79d299cfe21b5fa2602 INFO: Start
+ruby_app           | 2017-10-09T19:25:53.945Z 1 TID-grm1gy94g RubyApp::Workers::ProducerWorker JID-a1478d317c52a3f5c9306bec INFO: done: 0.002 sec
+ruby_app           | 2017-10-09T19:25:54.948Z 1 TID-grm1gy9dc RubyApp::Workers::ProducerWorker JID-b61c441b3799ab20fdc44ed8 INFO: start
+ruby_app           | 2017-10-09T19:25:54.949Z 1 TID-grm1gy9dc RubyApp::Workers::ProducerWorker JID-b61c441b3799ab20fdc44ed8 INFO: done: 0.001 sec
+crystal_app        | 2017-10-09T19:25:54.948Z 1 TID-j75vk  JID=4617f79d299cfe21b5fa2602 INFO: Done: 1.002594 sec
+crystal_app        | 2017-10-09T19:25:54.949Z 1 TID-j75s0  JID=a1d180f54753d9d7cf420d04 INFO: Start
+crystal_app        | 2017-10-09T19:26:03.951Z 1 TID-j75s0  JID=a1d180f54753d9d7cf420d04 INFO: Done: 9.001820 sec
+ruby_app           | 2017-10-09T19:26:03.951Z 1 TID-grm1gpmns RubyApp::Workers::ProducerWorker JID-7b8f207594d73c34c27db097 INFO: start
+crystal_app        | 2017-10-09T19:26:03.953Z 1 TID-j756o  JID=1128d1e593ef06e6a2379d0b INFO: Start
+ruby_app           | 2017-10-09T19:26:03.953Z 1 TID-grm1gpmns RubyApp::Workers::ProducerWorker JID-7b8f207594d73c34c27db097 INFO: done: 0.002 sec
+crystal_app        | 2017-10-09T19:26:04.955Z 1 TID-j756o  JID=1128d1e593ef06e6a2379d0b INFO: Done: 1.001937 sec
+ruby_app           | 2017-10-09T19:26:04.955Z 1 TID-grm1gy94g RubyApp::Workers::ProducerWorker JID-484f823dee4d311466e5e32e INFO: start
+crystal_app        | 2017-10-09T19:26:04.956Z 1 TID-j73sw  JID=886991c15c1a12ed3c6043a4 INFO: Start
+ruby_app           | 2017-10-09T19:26:04.957Z 1 TID-grm1gy94g RubyApp::Workers::ProducerWorker JID-484f823dee4d311466e5e32e INFO: done: 0.001 sec
 ```
 
 In the sample above you may see three distinct applications writing down:
-- crystal.1
-- ruby.1
-- tail
+- crystal_app
+- ruby_app
 
-The last one is going to print out every new line written into tmp/output.txt.
+you should use command `tail -f tmp/output.txt` to see every new line writed on tmp/output.txt
```

Comment:
Suggestion: "You may use `tail -f tmp/output.txt` to follow every new line written on the output file."